sbc_compassion/tools/import_letter_functions.py

```diff
@@ -21,13 +21,119 @@
 import checkboxreader as cbr
 import bluecornerfinder as bcf
 import numpy as np
+from math import ceil
 from glob import glob
 from wand.image import Image
+from collections import namedtuple
+
+from openerp import _
 
 
 ##########################################################################
 #                           GENERAL METHODS                              #
 ##########################################################################
+def testline(env, line_vals, csv_file_ids, doc_name):
+    """ Test each line and verify the data given by the csv.
+    A csv file need to be written in the same way than
+    sbc_compassion/tests/testdata/import/travis_file.csv
```

Comment:
No, it is not the same way. In travis_file, there is no 'barcode' column, but in this csv there should be one, otherwise the test fails.